src/shapes/IText/ITextClickBehavior.ts

```diff
@@ -10,12 +10,10 @@ import { ITextKeyBehavior } from './ITextKeyBehavior';
 import type { TOptions } from '../../typedefs';
 import type { TextProps, SerializedTextProps } from '../Text/Text';
 
-// TODO: this code seems wrong.
-// e.button for a left click is `0` and so different than `1` is more
-// not a right click. PR 3888 introduced this code and was about left clicks.
-function notALeftClick(e: MouseEvent) {
-  return e.button && e.button !== 1;
```

Comment:
because `e.button === 0` for a left click it would return `false`:
`notALeftClick(LEFT_CLICK_EV)` => `false` => is a left click 

so it was correct for left and right (`e.button === 2`) but not for middle click (`e.button === 1`) which would return false. Because it is rarly used we didn't catch it I think

It will account for event with no `button` property as before